envi/codeflow.py

```diff
@@ -154,10 +155,10 @@ def addCodeFlow(self, va, arch=envi.ARCH_DEFAULT):
             try:
                 op = self._mem.parseOpcode(va, arch=arch)
             except envi.InvalidInstruction, e:
-                print 'parseOpcode error at 0x%.8x: %s' % (va,e)
+                logging.getLogger("envi/codeflow.addCodeFlow").warning('parseOpcode error at 0x%.8x: %s', (va,e))
```

Comment:
this should not be a tuple, but have the values broken out as separate arguments.